Drop-In/src/main/java/com/braintreepayments/api/AddCardFragment.java

```diff
@@ -29,6 +29,8 @@ public class AddCardFragment extends DropInFragment implements OnCardFormSubmitL
     private AccessibleSupportedCardTypesView supportedCardTypesView;
     private AnimatedButtonView animatedButtonView;
 
+    private static Boolean cardLogosDisabled;
```

Comment:
```suggestion
    private Boolean cardLogosDisabled;
```

Reply:
It seemed like this needed to be static because the method `static AddCardFragment from(DropInRequest dropInRequest, @Nullable String cardNumber)` where we would have access to `dropInRequest` is static. Not sure if there is a better way to do this though?

Reply:
Yeah `static` would tie this to all instances of `AddCardFragment`. Parsing `DropInRequest` through the bundle args is the way to go.

Reply:
Nice! Will make that update. Writing this didn't feel right but I was like... idk what is happening because Java/Android.

Reply:
Yeah there's a million ways to do stuff in Android. That's the overall vibe haha.

Reply:
Updated: https://github.com/braintree/braintree-android-drop-in/pull/426/commits/93d402f65dc810560a216563871f2488c1cf9558. Thanks for the suggestion!